### DISCUSSION TITLE
是否考虑界面修改一下

### DISCUSSION THREAD

**Discussion by jiecaoshixiao** [hiroi-sora/Umi-OCR#503] [Ideas]
![ocr_画板 2](https://github.com/hiroi-sora/Umi-OCR/assets/54398025/e51323d0-b0c0-462c-81d3-2f7e9999d345)
![ocr_画板 3](https://github.com/hiroi-sora/Umi-OCR/assets/54398025/5074b410-8ed7-4345-9d9f-ee57a27d3657)

搞了一天，按照现在版本做了一点点界面优化，有很多不足。
可以留一个邮箱吗，我把源文件发过去，我希望能够实现，毕竟这是ocr中我认为比较好的。
为了隐私，也可以给我的邮箱发一条消息（zhangzhezhihua@163.com）
放心，免费，无版权，可以留证据，
但是这并不是我想要的效果，（因为效果不是很完善，毕竟是一天做出来的）
下图时我现在用的界面
![Snipaste_2024-05-05_19-55-59](https://github.com/hiroi-sora/Umi-OCR/assets/54398025/89449ca8-900a-4f22-ab32-012ba9596d3b)

我也知道前端代码设计制作的不容易，可能有些东西不好搞。
但我i希望做出一个更好的界面和使用体验

望采纳

不用担心这是免费软件等种种问题，我可以尽力而为，并且免费设计。

**Comment by hiroi-sora** [comment]
您好，感谢支持。

你可以将源文件打个压缩包，直接传到github这里哦。将文件拖进聊天框，等待上传完成即可。

**Comment by jiecaoshixiao** [comment]
[umi-OCR.zip](https://github.com/hiroi-sora/Umi-OCR/files/15223804/umi-OCR.zip)
我不会代码，不知道这个软件图标是代码写的还是图片，有什么问题可以问我，那个（ocr.ai）的文件就是源文件，用adobe Illustrator打开

**Reply by hiroi-sora** [reply]
感谢，我看了。虽然短期内不会做大幅修改，但是你的设计有参考价值，我会考虑。

**Reply by jiecaoshixiao** [reply]
[ocr暗色调.zip](https://github.com/hiroi-sora/Umi-OCR/files/15286199/ocr.zip)
好的
